<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUpNewTask</class>
 <widget class="QDialog" name="SetUpNewTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name of task:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewNameLineEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Description for task:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="NewDescriptionTextEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>291</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SavePushButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>390</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>290</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Final date:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Priority</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PriorityComboBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>LOW</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MEDIUM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HIGH</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="CancelPushButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="FinalDate">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>194</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
